plugins/node/opentelemetry-instrumentation-aws-lambda/README.md

```diff
@@ -86,6 +85,34 @@ The `lambdaHandler` should be specified as a string in the format `<file>.<handl
 
 One way to determine if the `lambdaHandler` option should be used is to check the handler defined on your Lambda. This can be done by determining the value of the `_HANDLER` environment variable or by viewing the **Runtime Settings** of your Lambda in AWS Console. If the handler is what you expect, then the instrumentation should work without the `lambdaHandler` option. If the handler points to something else, then the `lambdaHandler` option should be used to explicitly specify the handler that should be instrumented.
 
+### Active Tracing Context Propagation
```

Comment:
This section describes when one should use the `xray-lambda` propagator.
Should we also have a section for the `xray` propagator with examples on when and how to set it up? The spec mention it and I think it can be useful to users looking for instructions.

Reply:
I changed the section to a more general `Context Propagation`, and added a section that talks about the difference between `xray-lambda` and `xray`.